pkg/crawler/pomutils.go

```diff
@@ -89,12 +89,12 @@ func parseAndSubstitutePom(url string) (PomProject, error) {
 	var project PomProject
 
 	resp, err := http.Get(url)
-	if resp.StatusCode == http.StatusNotFound {
-		return project, nil
-	}
 	if err != nil {
 		return project, xerrors.Errorf("can't get pom xml from %s: %w", url, err)
 	}
+	if resp.StatusCode == http.StatusNotFound {
```

Comment:
@namandf Wasn't there a reason this was kept before the error check?

Reply:
Hmm. Can't recall. But Hrithik's change seems like the right thing to do.

Reply:
@hrithik-777  you can probably double check the behaviour in case of 404 to be sure